classes/Product.php

```diff
@@ -5516,7 +5516,7 @@ public static function getProductProperties($id_lang, $row, Context $context = n
         $row['category'] = Category::getLinkRewrite((int) $row['id_category_default'], (int) $id_lang);
         $row['category_name'] = Db::getInstance()->getValue('SELECT name FROM ' . _DB_PREFIX_ . 'category_lang WHERE id_shop = ' . (int) $context->shop->id . ' AND id_lang = ' . (int) $id_lang . ' AND id_category = ' . (int) $row['id_category_default']);
         $row['link'] = $context->link->getProductLink((int) $row['id_product'], $row['link_rewrite'], $row['category'], $row['ean13']);
-        $row['manufacturer_name'] = (int) $row['id_manufacturer'] > 0 ? Manufacturer::getNameById((int) $row['id_manufacturer']) : null;
+        $row['manufacturer_name'] = !empty($row['id_manufacturer']) ? Manufacturer::getNameById((int) $row['id_manufacturer']) : null;
```

Comment:
I'm not sure about what you are fixing here

```
(int) '' === 0
(int) false === 0
(int) null === 0
``` 

So if I'm not wrong `(!empty($row['id_manufacturer']))` and `(int) $row['id_manufacturer'] > 0` are the same :/ (if `$row['id_manufacturer']` exists)

Unless i'm missing something

Reply:
The problem is that this key doesn't exist in cart context. `id_manufacturer` must be present in the array for casting to work.

Reply:
That is a `empty` behaviour i was not aware of (and it confirms for me that `empty` is 🤢).
 Fix is good for me

Reply:
`empty` is amazing! :D Oneline check for everything, in most scenarios.